README_DOCKERHUB.tpl.md

```diff
@@ -6,7 +6,9 @@ There are two types of images available: images with [sonar-scala](https://githu
 
 Starting from version `2.7.0`, the images no longer contain the [sonar-scala-extra](https://github.com/arthepsy/sonar-scala-extra) plugin as sonar-scala provides Scapegoat support from version `6.5.0` onwards.
 
-- `3.2.1`, `latest` [Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/3.2.1/Dockerfile), [(v3.2.1)](https://github.com/mwz/sonar-scala-docker/releases/tag/3.2.1)
+- `3.3.0`, `latest` [Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/3.3.0/Dockerfile), [(v3.3.0)](https://github.com/mwz/sonar-scala-docker/releases/tag/3.3.0)
+- `3.3.0-full` [Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/3.3.0/Dockerfile), [(v3.3.0)](https://github.com/mwz/sonar-scala-docker/releases/tag/3.3.0)
```

Comment:
Here the Dockerfile path is incorrect. You are pointing to the plain `3.3.0` Dockerfile instead of the `3.3.0-full` Dockerfile.

Reply:
Thanks for spotting this! Maintaining this list is quite error-prone, I should probably do something to make it easier... 🤔

Reply:
That's a good idea and it can be totally done with mustache - I'll make the improvement in a new PR.